dist/version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "21.2.0"
+  "version": "21.1.4"
```

Comment:
Not sure if we should revert this or bump it again?

Reply:
I think we would have to "fail foward"

Reply:
👍